packages/story-editor/src/output/page.js

```diff
@@ -81,15 +83,25 @@ function OutputPage({ page, autoAdvance = true, defaultPageDuration = 7 }) {
 
   const videoCaptions = elements
     .filter(
-      ({ type, tracks }) => type === ELEMENT_TYPES.VIDEO && tracks?.length > 0
+      ({ type, tracks: videoTracks }) =>
+        type === ELEMENT_TYPES.VIDEO && videoTracks?.length > 0
     )
     .map(({ id: videoId }) => `el-${videoId}-captions`);
 
+  const hasBackgroundAudioWithTracks =
+    backgroundAudio?.src && tracks?.length > 0;
+
+  if (hasBackgroundAudioWithTracks) {
+    videoCaptions.push(`el-${backgroundAudio.id}-captions`);
+  }
+
+  const backgroundAudioSrc =
+    backgroundAudio?.src && !tracks?.length ? backgroundAudio.src : undefined;
```

Comment:
It would be nice to re-use `hasBackgroundAudioWithTracks` here for improved readability

```js
const backgroundAudioSrc = !hasBackgroundAudioWithTracks && backgroundAudio?.src ? backgroundAudio?.src : undefined;
```

Reply:
Fixed in https://github.com/GoogleForCreators/web-stories-wp/pull/10288/commits/c4702cdff4a2b35dc06052d7e3d0c6898bd4696d